tools/tests/tester/fixtures/stories/passed.yaml

```diff
@@ -44,3 +44,13 @@ chapters:
       content_type: application/cbor
       payload:
         - node.total: '1'
+  - synopsis: This GET /_cat/health chapter returns application/smile and should pass.
+    path: /_cat/health
+    parameters:
+      format: cbor
+    method: GET
+    response:
+      status: 200
+      content_type: application/smile
```

Comment:
As above regarding validating `content_type` as test is supposed to be for `smile` but format is set to `cbor`

Reply:
Updated to be `format: smile`.